purchase_landed_cost/models/purchase_cost_distribution.py

```diff
@@ -562,12 +562,26 @@ def _compute_display_name(self):
     @api.onchange('type')
     def onchange_type(self):
         if self.type and self.type.default_amount:
-            self.expense_amount = self.type.default_amount
+            # set expense_amount in the currency of the distribution
```

Comment:
Good text self-explanatory but might be better as the docstring of the function?

Reply:
Done